public/video-ui/src/components/Video/Display.js

```diff
@@ -207,11 +208,14 @@ class VideoDisplay extends React.Component {
   }
 
   renderSelectBar(video) {
+
+    const videoToSelect = isVideoPublished(this.props.publishedVideo) ?
```

Comment:
this is a weird statement `isVideoPublished(this.props.publishedVideo)` 😂

Reply:
Yeah, makes more sense to pass in `this.props.video` here.